cylc/flow/network/schema.py

```diff
@@ -2177,7 +2190,7 @@ class Mutations(ObjectType):
     poll = _mut_field(Poll)
     release = _mut_field(Release)
     remove = _mut_field(Remove)
-    set_outputs = _mut_field(SetOutputs)
+    set_prereqs_and_outputs = _mut_field(SetPrereqsAndOutputs)
```

Comment:
This is the public-facing name of the mutation, i.e. what you would call from GraphQL and what the command will be called in cylc-ui:

```suggestion
    set = _mut_field(SetPrereqsAndOutputs)
```

Reply:
I thought decided it was OK as set, but as we could go either way it was still better to avoid the stdlib name.
Probably my misunderstanding, I'll change it back.

Reply:
I don't think this one got pushed?

```diff
diff --git a/cylc/flow/network/schema.py b/cylc/flow/network/schema.py
index 77ed81147..a99183d22 100644
--- a/cylc/flow/network/schema.py
+++ b/cylc/flow/network/schema.py
@@ -2190,7 +2190,7 @@ class Mutations(ObjectType):
     poll = _mut_field(Poll)
     release = _mut_field(Release)
     remove = _mut_field(Remove)
-    set_prereqs_and_outputs = _mut_field(SetPrereqsAndOutputs)
+    set = _mut_field(SetPrereqsAndOutputs)
     trigger = _mut_field(Trigger)
 
     # job actions
diff --git a/cylc/flow/scripts/set.py b/cylc/flow/scripts/set.py
index e3e0525a2..a09cf3396 100755
--- a/cylc/flow/scripts/set.py
+++ b/cylc/flow/scripts/set.py
@@ -112,7 +112,7 @@ mutation (
   $flowWait: Boolean,
   $flowDescr: String,
 ) {
-  setPrereqsAndOutputs (
+  set (
     workflows: $wFlows,
     tasks: $tasks,
     prerequisites: $prerequisites,
```